logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/selfdeletingMessages/PersistNewSelfDeletionTimerUseCase.kt

```diff
@@ -37,6 +37,14 @@ class PersistNewSelfDeletionTimerUseCaseImpl(
 ) : PersistNewSelfDeletionTimerUseCase {
     override fun invoke(conversationId: ConversationId, newSelfDeletionTimer: SelfDeletionTimer) =
         userConfigRepository.setConversationSelfDeletionTimer(conversationId, newSelfDeletionTimer).fold({
-            kaliumLogger.e("Failure while persisting self deleting messages status $it")
-        }, { kaliumLogger.d("Successfully updated self deleting messages status $newSelfDeletionTimer") })
+            kaliumLogger.e(
+                "Failure while persisting a new Self Deletion Timer for conversation=${conversationId.toLogString()} with " +
```

Comment:
This is not structured, the whole event must be represented as json

`Self Deleting Failure: {json-of-the-event-to-log}`